source/imaer-gml/src/main/java/nl/overheid/aerius/gml/v4_0/metadata/MetaDataImpl.java

```diff
@@ -127,7 +127,7 @@ public String getSituationName() {
 
   @Override
   public SituationType getSituationType() {
-    return situation == null ? null : situation.getSituationType();
+    return situation.getSituationType().toSituationType();
```

Comment:
Might want to leave the null checks in here, SituationMetaData is not a required field.